files/system/usr/share/ublue-os/just/70-secureblue.just

```diff
@@ -407,18 +407,17 @@ audit-secureblue:
     fi
 
     MODPROBE_TEST_STRING="Ensuring no modprobe overrides"
-    if diff /usr/etc/modprobe.d/blacklist.conf /etc/modprobe.d/blacklist.conf > /dev/null; then
+    readarray -t unwanted_modules < <(comm -12 <(lsmod | cut -f 1 -d " " | sort) <(cat /usr/etc/modprobe.d/blacklist.conf | grep -E '^(blacklist)|(install)' | cut -f 2 -d " " | sort))
+    if [[ -z "${#modules[@]}" ]] && diff /usr/etc/modprobe.d/blacklist.conf /etc/modprobe.d/blacklist.conf > /dev/null; then
```

Comment:
where is the modules variable getting populated?

Reply:
fixed, thank you